content/en/blog/_posts/2024-05-09-devsecops-in-action-k8s-secrets/index.md

```diff
@@ -0,0 +1,554 @@
+---
+layout: blog
+title: "DevSecOps in Action: Kubernetes Secrets"
+slug: devsecops-in-action-k8s-secrets
+date: 2024-05-09
+author: >
+  Evgenii Frikin
+---
+
+
+The Secret is one of the basic resources that exists in the Kubernetes API. This
+kind of resource is used for different purposes and despite the name Secret,
+sensitive data is insecure to store in it. Recently considerable attention has
+been paid to
+[security improvement](/docs/tasks/administer-cluster/encrypt-data/)
+for kind of Secret resource, but Kubernetes still doesn't include appropriate
+out-of-the-box because Kubernetes philosophy carries simplicity in order not to
+compromise it engineers have to make sure Kubernetes Secret are saved
```

Comment:
PTAL at the suggestion for better clarity.

```suggestion
Recently, considerable attention has been paid to [security improvement](/docs/tasks/administer-cluster/encrypt-data/) for the Secret resource, but Kubernetes still doesn't include appropriate out-of-the-box solutions. And because the Kubernetes philosophy emphasizes simplicity, the engineers have to ensure that Kubernetes Secrets are securely saved.
```